torchgeo/datasets/l8biome.py

```diff
@@ -0,0 +1,225 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""L8Biome dataset."""
+
+import glob
+import os
+from typing import Any, Callable, Dict, List, Optional, cast
+
+from rasterio.crs import CRS
+
+from .geo import RasterDataset
+from .utils import BoundingBox, download_url, extract_archive
+
+
+class L8Biome(RasterDataset):
+    r"""L8 Biome datasets.
+
+    The `L8 Biome <https://landsat.usgs.gov/landsat-8-cloud-cover-assessment-validation-data>`__ dataset
+    is a cloud validation dataset of Pre-Collection Landsat 8 Operational Land Imager (OLI) Thermal Infrared Sensor (TIRS) terrain-corrected (Level-1T) scenes.
+
+    Dataset features:
+    * images evenly divided between eight unique biomes
+    * 5 cloud cover categories
+
+    Dataset format:
+
+    * Each cloud mask is in ENVI binary format.
+    Includes all bands from the original Landsat Level-1 data product (GeoTIFF), and its associated Level-1 metadata (MTL.txt file)
+
+    If you use this dataset in your research, please cite the following:
+
+    * https://doi.org/10.5066/F7251GDH
+    * https://doi.org/10.1016/j.rse.2017.03.026
+
+    .. versionadded:: 0.5
+    """
+
+    url = "https://huggingface.co/datasets/torchgeo/l8biome/blob/main/{}.tar.gz"
+    # redistributed from https://landsat.usgs.gov/landsat-8-cloud-cover-assessment-validation-data
```

Comment:
This is documented on Hugging Face, and the original homepage is already in the docstring, so I would probably remove this comment.